sim/midas/src/test/scala/midas/core/SimUtilsSpec.scala

```diff
@@ -67,7 +74,7 @@ class SimUtilsSpec extends AnyFlatSpec {
         portIntRef.copy(component = Seq(TargetToken.Field("bits")))
       ),
     )
-    require(data.typeEquivalent(UInt(32.W)))
+    require(DataMirror.checkTypeEquivalence(data, UInt(32.W)))
```

Comment:
Why did you have to change these? I feel like nandor got them already?

Reply:
I changed these by grabbing from nandor's commit directly because his commit was waiting to be merge to master. I could have waited and rebased instead

Reply:
If you want, I can rebase before we merge. However because I pulled the files exactly the squash merge should identify the files as same and not generate any conflicts